README.md

```diff
@@ -1,2 +1,15 @@
 # ExpenseTrackerGrupo5
 Take control of your finances.
+
+
+### Levantar contenedores 
```

Comment:
please, write the documentation in English

Reply:
Oh I forget that give a minuto to fix that